AtlasStarter/Program.cs

```diff
@@ -0,0 +1,185 @@
+
+            var mongoUri = "";
+
+            // The IMongoClient is the object that defines the connection to our
+            // datastore (Atlas, for example)
+            IMongoClient client;
+
+            // An IMongoCollection defines a connection to a specific MongoDB
+            // collection. Your app may have one or many different IMongoCollection
+            // objects.
+            IMongoCollection<Recipe> collection;
+
+            // Note that you must define the *type* of data stored in the
+            // IMongoCollection. We have created a class called TestDocument at
+            // the bottom of this file that serves as a "mapping class" -- the 
+            // driver maps the C# class to the BSON stored in MongoDB.
+
+            // Using mapping classes is strongly advised, but if you
+            // don't create them, you can always use the more generic BsonDocument
+            // type.
+
+            try
+            {
+                client = new MongoClient(mongoUri);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            // Provide the name of the database and collection you want to use.
+            // If they don't already exist, the driver and Atlas will create them
+            // automatically when you first write data.
+            var dbName = "testDatabase";
+            var collectionName = "testCollection";
+
+            try
+            {
+                collection = client.GetDatabase(dbName)
+                    .GetCollection<Recipe>(collectionName);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** INSERT DOCUMENTS ***
+             * 
+             * You can insert individual documents using collection.Insert(). 
+             * In this example, we're going to create 4 documents and then 
+             * insert them all in one call with InsertMany().
+             */
+
+            var docs = Recipe.GetRecipes();
+
+            try
+            {
+                collection.InsertMany(docs);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** FIND DOCUMENTS ***
+             * 
+             * Now that we have data in Atlas, we can read it. To retrieve all of
+             * the data in a collection, we call Find() with an empty filter. 
+             * The Builders class is very helpful when building complex 
+             * filters, and is used here to show its most basic use.
+             */
+
+            var allDocs = collection.Find(Builders<Recipe>.Filter.Empty)
+                .ToList();
+
+            foreach (Recipe doc in allDocs)
+            {
+                // Do something with each document ...
```

Comment:
Printing all the recipes here after a statement like 'Found N documents in the collection" might make this easier to follow and run.